persistence/sql/migrations/sql/20240923095000000001_organization_id_index.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX identities_nid_organization_id_idx ON identities (nid, organization_id);
```

Comment:
This should be `... identities (organization_id,created_at);`

`organization_id` is sharded by NID anyway, so we can reduct index size here.

Reply:
Maybe `(organization_id,id)`. I thought we list be created date?

Reply:
We might need an index hint here, too.

Reply:
indices sort their keys anyways, so no need to add id for sorting to the index